src/drivers/pwm_input/pwm_input.cpp

```diff
@@ -294,6 +293,12 @@ PWMIN::~PWMIN()
  * to be started in init scripts when the user may be using the input
  * pin as PWM output
  */
+
+static void _freeze_test()
```

Comment:
Please rename to `freeze_test_trampoline(void *arg)`, use `arg` instead of `g_dev`, and rename `PWMIN::_turn_on` to `PWMIN::turn_on`, since it's public now.

Also the comment above (`initialze`...) is not next to `PWMIN::init()` anymore.